### PR TITLE
Fixing the 'BV' key being missing for USDT

### PR DESCRIPTION
**Pull request by jblestang**
## Summary
This PR fixes handling of data fetched from Bittrex server with missing Keys (BV is not present in USDT_BTC)

Solve the issue: #397  

## Quick changelog

- dropping of the 'BV' column is made only if the column exists

### CODE REVIEW

File: freqtrade/tests/conftest.py [hunk 1]
```diff
@@ -195,7 +195,6 @@ def ticker_history():
             "C": 8.88e-05,
             "V": 991.09056638,
             "T": "2017-11-26T08:50:00",
-            "BV": 0.0877869
```
Comment (shusso) [inline freqtrade/tests/conftest.py]: Shouldn't we have some of the test data with and without `BV` flag? So we know that we cover both cases?

Comment (kryofly) [reply freqtrade/tests/conftest.py]: I thought that error was triggered, when getting an empty result back. Didn't know bittrex could be so nasty.

Comment (jblestang) [reply freqtrade/tests/conftest.py]: That is not what the backtrace shows. The problem you're referring has been solved in another PR #411

Comment (glonlas) [reply freqtrade/tests/conftest.py]: I will help to forge one and send another PR. Between I think we should merge this PR.

Comment (glonlas) [reply freqtrade/tests/conftest.py]: More I am thinking to this more I get aligned to @shusso.
@jblestang could you add two set of tickets: with and without BV like below:

```python
@pytest.fixture
def ticker_history():
    return [
        {
            "O": 8.794e-05,
            "H": 8.948e-05,
            "L": 8.794e-05,
            "C": 8.88e-05,
            "V": 991.09056638,
            "T": "2017-11-26T08:50:00",
            "BV": 0.0877869
        },
        {
            "O": 8.88e-05,
            "H": 8.942e-05,
            "L": 8.88e-05,
            "C": 8.893e-05,
            "V": 658.77935965,
            "T": "2017-11-26T08:55:00",
            "BV": 0.05874751
        },
        {
            "O": 8.891e-05,
            "H": 8.893e-05,
            "L": 8.875e-05,
            "C": 8.877e-05,
            "V": 7920.73570705,
            "T": "2017-11-26T09:00:00",
            "BV": 0.7039405
        }
    ]


@pytest.fixture
def ticker_history_without_bv():
    return [
        {
            "O": 8.794e-05,
            "H": 8.948e-05,
            "L": 8.794e-05,
            "C": 8.88e-05,
            "V": 991.09056638,
            "T": "2017-11-26T08:50:00"
        },
        {
            "O": 8.88e-05,
            "H": 8.942e-05,
            "L": 8.88e-05,
            "C": 8.893e-05,
            "V": 658.77935965,
            "T": "2017-11-26T08:55:00"
        },
        {
            "O": 8.891e-05,
            "H": 8.893e-05,
            "L": 8.875e-05,
            "C": 8.877e-05,
            "V": 7920.73570705,
            "T": "2017-11-26T09:00:00"
        }
    ]
```

And in `test/test_analyse.py`

```python
def test_parse_ticker_dataframe(ticker_history, ticker_history_without_bv):

    columns = ['close', 'high', 'low', 'open', 'date', 'volume']

    # Test file with BV data
    dataframe = parse_ticker_dataframe(ticker_history)
    assert dataframe.columns.tolist() == columns

    # Test file without BV data
    dataframe = parse_ticker_dataframe(ticker_history_without_bv)
    assert dataframe.columns.tolist() == columns

```

Comment (glonlas) [reply freqtrade/tests/conftest.py]: More I am thinking to this more I get aligned to @shusso.
@jblestang could you add two set of tickets: with and without BV like below:

```python
@pytest.fixture
def ticker_history():
    return [
        {
            "O": 8.794e-05,
            "H": 8.948e-05,
            "L": 8.794e-05,
            "C": 8.88e-05,
            "V": 991.09056638,
            "T": "2017-11-26T08:50:00",
            "BV": 0.0877869
        },
        {
            "O": 8.88e-05,
            "H": 8.942e-05,
            "L": 8.88e-05,
            "C": 8.893e-05,
            "V": 658.77935965,
            "T": "2017-11-26T08:55:00",
            "BV": 0.05874751
        },
        {
            "O": 8.891e-05,
            "H": 8.893e-05,
            "L": 8.875e-05,
            "C": 8.877e-05,
            "V": 7920.73570705,
            "T": "2017-11-26T09:00:00",
            "BV": 0.7039405
        }
    ]


@pytest.fixture
def ticker_history_without_bv():
    return [
        {
            "O": 8.794e-05,
            "H": 8.948e-05,
            "L": 8.794e-05,
            "C": 8.88e-05,
            "V": 991.09056638,
            "T": "2017-11-26T08:50:00"
        },
        {
            "O": 8.88e-05,
            "H": 8.942e-05,
            "L": 8.88e-05,
            "C": 8.893e-05,
            "V": 658.77935965,
            "T": "2017-11-26T08:55:00"
        },
        {
            "O": 8.891e-05,
            "H": 8.893e-05,
            "L": 8.875e-05,
            "C": 8.877e-05,
            "V": 7920.73570705,
            "T": "2017-11-26T09:00:00"
        }
    ]
```

And in `test/test_analyse.py`

```python
def test_parse_ticker_dataframe(ticker_history, ticker_history_without_bv):

    columns = ['close', 'high', 'low', 'open', 'date', 'volume']

    # Test file with BV data
    dataframe = parse_ticker_dataframe(ticker_history)
    assert dataframe.columns.tolist() == columns

    # Test file without BV data
    dataframe = parse_ticker_dataframe(ticker_history_without_bv)
    assert dataframe.columns.tolist() == columns

```

**Comment by jblestang** [comment]
Seems like an honest request!

**Comment by jblestang** [comment]
I've messed up the branch, new PR #417 created with a cleaner history

**Pull request closed without merging**